pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -799,7 +799,7 @@ func (r *ReconcileInfinispan) statefulSetForInfinispan(m *infinispanv1.Infinispa
 		},
 			Spec: corev1.PersistentVolumeClaimSpec{
 				AccessModes: []corev1.PersistentVolumeAccessMode{
-					"ReadWriteOnce",
+					corev1.ReadWriteOnce,
```

Comment:
Is this intended?
Change is ok by me, better to align #593 also?

Reply:
Yes, @rigazilla. I know that would be better to create a separate issue for memories and fix all non-related things, but didn't this yet.
Will do for #593 also